src/Components/test/testassets/Components.TestServer/RazorComponents/Pages/Routing/Encoded/EncodedCatchAll.razor

```diff
@@ -0,0 +1,25 @@
+﻿@page "/🙂/routing/catch-all/{*parameter}"
```

Comment:
If I ever see a URL like this in the wild, I'll know it's time to go live off-grid and pretend never to have heard of computers...